packages/mask/src/utils/components/WalletStatusBar/WalletMenuItem.tsx

```diff
@@ -55,8 +66,16 @@ export const WalletMenuItem = memo<WalletMenuItemProps>(
 
         return (
             <MenuItem value={address} onClick={() => onSelect?.(descriptionProps, chainId, pluginId)}>
+                {/* TODO: replace to radio */}
                 <ListItemIcon>
-                    <Radio checked={selected} />
+                    {selected ? (
+                        <Icons.RadioButtonChecked
+                            className={classes.icon}
+                            style={{ filter: 'drop-shadow(0px 0px 6px rgba(28, 104, 243, 0.6))' }}
+                        />
+                    ) : (
+                        <Icons.RadioButtonUnChecked className={classes.icon} />
+                    )}
```

Comment:
Why not just control with size?

```suggestion
                        <Icons.RadioButtonChecked
                            size={24}
                            style={{ filter: 'drop-shadow(0px 0px 6px rgba(28, 104, 243, 0.6))' }}
                        />
                    ) : (
                        <Icons.RadioButtonUnChecked size={24} />
                    )}
```

Reply:
There are two different icons.